src/API/interface/Parameters.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\API\Int;
+
+/**
+ * Parameters
+ *
+ * @author killing@leifos.de
+ */
+interface Parameters
```

Comment:
I am not a friend of empty interfaces just for the purpose of having created such :-). But of course this is an approach/prototyp and maybe we need this kind of interface later.